src/test/java/com/wikia/webdriver/common/driverprovider/DriverProvider.java

```diff
@@ -40,8 +40,11 @@ public static void close() {
     for (WikiaWebDriver webDriver : drivers) {
       if (webDriver != null) {
         try {
+          String path = System.getenv("PATH");
+          System.out.println(path);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS106)